cfgov/agreements/jinja2/agreements/index.html

```diff
@@ -50,11 +50,20 @@ <h2 id="issuer-select">
                 </p>
 
                 <ul class="m-list m-list__links u-mt15">
+                    <li class="m-list m-list__links">
+                        <a class="a-link a-link__icon"
+                        href="https://files.consumerfinance.gov/a/assets/Credit_Card_Agreements_2019_Q1.zip">
+                            <span class="a-link_text">
+                                Download all most recent agreements (Q1-2019)
+                            </span>
```

Comment:
Sorry I'm too late to this, but if you remove the whitespace inside the `<span>` tags, you'll get rid of the underline going under the space between the text and the icon.